recipes/scipy/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version="0.17.0" %}
+
+package:
+  name: scipy
+  version: {{ version }}
+
+source:
+  fn: scipy-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scipy/scipy-0.17.0.tar.gz
+  md5: 5ff2971e1ce90e762c59d2cd84837224
+
+build:
+  number: 0
+  features:
+    - openblas
```

Comment:
Should we name it something else like `blas_openblas`? Naming it the same as the actual package has caused me issues in the past.

cc @mcg1969